autopush/main.py

```diff
@@ -30,10 +27,9 @@
     DefaultResource,
     StatusResource,
 )
-from autopush.web.simplepush import SimplePushHandler
-from autopush.web.webpush import WebPushHandler
-from autopush.web.limitedhttpconnection import LimitedHTTPConnection
-
+from twisted.internet import reactor, task
+from twisted.logger import Logger
+from twisted.web.server import Site
```

Comment:
This looks like pycharm may have helped. flake8 also didn't gripe, obviously. 

I'll reshuffle these to our preferred ordering and grouping.